test/test_expression.py

```diff
@@ -537,6 +537,23 @@ def test_complex_support(ctx_factory, target):
     np.testing.assert_allclose(out["tmp_sum"], (0.5*n*(n-1) + 0.5*n*(n-1)*1j) ** 2)
 
 
+def test_bool_type_context(ctx_factory):
```

Comment:
Add a docstring to say what used to fail.

Reply:
Sure ff016d5.